prodtests/full-system-test/workflow-setup.sh

```diff
@@ -13,14 +13,15 @@ else
   if [[ -z "${WORKFLOW_DETECTORS_MATCHING+x}" ]]; then export WORKFLOW_DETECTORS_MATCHING="ALL"; fi # All matching / vertexing enabled in async mode
 fi
 
-[[ -z $ITS_STROBE ]] && ITS_STROBE="891"
-[[ -z $MFT_STROBE ]] && MFT_STROBE="198"
+if [[ $BEAMTYPE == "PbPb" ]]; then
+  [[ -z $ITS_STROBE ]] && ITS_STROBE="891"
+fi
```

Comment:
if you don't want to depend on what CCDB will supply, is not it better to set it to a specific value also for pp?

Reply:
hm, ok, we can do that